### ISSUE TITLE
Intermittent errors with OpenMM 7.5.1 benchmarks

### ISSUE THREAD

**Issue by dmclark17** [openmm/openmm#3170]
I am noticing some intermittent errors when running OpenMM 7.5.1 benchmarks; specifically, with the default timestep, the pme test sometimes throws “Particle coordinate is nan” exceptions. It does seem to be quite rare—the error happened in two of 100 trials. This was done on a Titan V and CUDA 11.2. I am using the following benchmark command:

```
python benchmark.py --test=pme --platform=CUDA --seconds=30
```

Thanks!

**Comment by peastman** [comment]
Thanks!  My guess is that we just need to add `hydrogenMass = 1.5*unit.amu` for DHFR, the same way we do for ApoA1.

**Comment by dmclark17** [comment]
Sorry for the delay. Increasing the hydrogen mass seemed to fix the issue—the benchmark successfully finished all of the 100 trials that I ran. I can submit a PR with the change.

**Comment by peastman** [comment]
That would be great.  Let's just make that our standard for most benchmarks.  The logic would be:

```python
if amoeba:
  hydrogenMass = None
elif options.heavy:
  hydrogenMass = 4*unit.amu
else:
  hydrogenMass = 1.5*unit.amu
```

**Comment by peastman** [comment]
Fixed by #3208.
